cmd/destroy/local.go

```diff
@@ -16,300 +16,75 @@ package destroy
 import (
 	"context"
 	"errors"
-	"fmt"
```

Comment:
The logic removed from here was moved to 2 places:
* Most of them were moved to `destroy/destroy.go` as is logic to be executed always locally
* Other part was moved to the `deployable/destroy.go` as that is the logic that has to be executed locally or remotely and it is shared logic